structs.go

```diff
@@ -745,6 +746,9 @@ type Guild struct {
 
 	// Permissions of our user
 	Permissions int64 `json:"permissions,string"`
+
+	// StageInstance in the guild
```

Comment:
```suggestion
	// Stage instances in the guild
```

Reply:
I will do this changes to follow current conventions of files. In the future follow the comment convention of Golang : All commented function, struct or field must be start with the name of the element